spec/deck_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+require 'deck'
+
+describe Deck do
+  before(:each) do
+    @deck = Deck.new
+  end
+
+  it 'initializes with 52 cards' do
+    @deck.cards.size.must_equal 52
+  end
+
+  it 'responds to a shuffle method' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.